swagger.yaml

```
@@ -805,6 +898,8 @@ paths:
 responses:
   InternalError:
     description: "Failed to process the request due to an internal error"
+  UnauthorizedError:
+    description: Authentication information is missing or invalid
```

Should these responses have a response body (i.e. schema) or do they have an empty body?

---

it's an empty body

---

That is probably a defect we should log somewhere as all errors should return an error message body.